core/src/main/scala/kafka/admin/AdminClient.scala

```diff
@@ -114,6 +116,13 @@ class AdminClient(val time: Time,
     listAllGroupsFlattened.filter(_.protocolType == ConsumerProtocol.PROTOCOL_TYPE)
   }
 
+  def listGroupOffsets(groupId: String): Map[TopicPartition, PartitionData] = {
+    val coordinator = findCoordinator(groupId)
+    val responseBody = send(coordinator, ApiKeys.OFFSET_FETCH, OffsetFetchRequest.allPartitionsOffsetFetchRequest(groupId))
+    val response = responseBody.asInstanceOf[OffsetFetchResponse]
+    response.responseData().asScala.toMap
```

Comment:
@hachikuji One more question before I re-open the KIP. The issue doesn't seem to be a side-effect of the suggested protocol change in KIP-88 (we are not modifying the response in KIP-88) and it would surface whenever there is no partition in the response. Do you think it can be addressed in a separate KIP? Or I am missing something here?